src/components/ForestTypeModal/ForestTypeComparison/bl/ForestTypeTab.js

```diff
@@ -0,0 +1,318 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { Table } from 'semantic-ui-react';
+import { info } from '@geops/tree-lib';
+import useIsMobile from '../../../../hooks/useIsMobile';
+
+import ComparisonCell from '../ComparisonCell';
+import ForestTypeLink from '../../ForestTypeLink';
+import ForestTypeLinksList from '../../ForestTypeLinksList';
+import Site from '../../ForestTypeDescription/bl/Site';
+import {
+  setForestTypeDescription,
+  setForestTypeModal,
+} from '../../../../store/actions';
+import {
+  treeTypeMapping,
+  vegetationMapping,
+} from '../../ForestTypeDescription/bl/utils';
+import SoilIcon from '../../../../icons/SoilIcon';
+import comparisonStyles from '../ForestTypeComparison.module.css';
+
+const getTreeTypes = (treeTypes, category) =>
+  treeTypeMapping.reduce(
+    (finalTreeTypes, treeType, index) =>
+      treeTypes[index] === category
+        ? `${finalTreeTypes}${`${finalTreeTypes ? ', ' : ''}${treeType}`}`
+        : finalTreeTypes,
+    '',
+  );
+
+const HeaderCell = ({ ...props }) => {
+  const { children } = props;
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <Table.HeaderCell {...props} verticalAlign="top">
+      {children}
+    </Table.HeaderCell>
+  );
+};
+HeaderCell.propTypes = {
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+};
+
+function ForestTypeTab({ data }) {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const isMobile = useIsMobile();
+
+  return (
+    <Table basic padded structured fixed>
+      <Table.Body>
+        {!isMobile && (
+          <Table.Row>
+            <HeaderCell>Standortstyp</HeaderCell>
+            {data.map((ft) => (
+              <HeaderCell key={ft.code}>
+                <ForestTypeLink code={ft.code} />
+              </HeaderCell>
+            ))}
+          </Table.Row>
+        )}
+        <Table.Row>
+          <HeaderCell>Lateinische Bezeichnung</HeaderCell>
+          {data.map((ft, idx, arr) => (
+            <ComparisonCell
+              key={ft.code}
+              code={ft.code}
+              data={ft.la}
+              footer={isMobile && idx + 1 !== arr.length && <br />}
+            />
+          ))}
+        </Table.Row>
+        <Table.Row className={comparisonStyles.row}>
+          <HeaderCell>Eigenschaften</HeaderCell>
+          {data.map((ft, idx, arr) => (
+            <ComparisonCell
+              key={ft.code}
+              code={ft.code}
+              data={ft.properties}
+              footer={isMobile && idx + 1 !== arr.length && <br />}
+            />
+          ))}
+        </Table.Row>
+
+        {/* {treeTypeCells.map((tt, idx, arr) => (
```

Comment:
Please cleanup.

Reply:
This was for the tillering, so I deliberately left it there as a TODO. It's implemented now